examples/gnn_node.py

```diff
@@ -168,25 +166,44 @@ def test(loader: NeighborLoader) -> np.ndarray:
     return torch.cat(pred_list, dim=0).numpy()
 
 
-state_dict = None
-best_val_metric = 0 if higher_is_better else math.inf
-for epoch in range(1, args.epochs + 1):
-    train_loss = train()
+state_dicts = []
+
+for ensemble_idx in range(args.num_ensembles):
+    print(f"===Training for {ensemble_idx}-th ensemble index.")
+    model.reset_parameters()
+    optimizer = torch.optim.Adam(model.parameters(), lr=args.lr)
```

Comment:
personally would prefer to just instantiate new models. Allows you also to do a hyperparam sweep if you want.

Reply:
Sounds good.